plugins/color_panel/lib/color_panel.dart

```diff
@@ -55,14 +56,15 @@ class ColorPanel {
   ///
   /// [callback] will be called when the user selects a color; it can be called
   /// an number of times depending on the interaction model of the native
-  /// panel.
+  /// panel. Set [showAlphaSlider] to true to show an opacity slider.
   ///
   /// It is an error to call [show] if the panel is already showing.
-  void show(ColorPanelCallback callback) {
+  void show(ColorPanelCallback callback, {bool showAlphaSlider}) {
```

Comment:
I'd like the API (comment, variable name, string constant) to not express expectations about the exact UI. There's no guarantee that any given platform implementation will use a slider rather than, say, just a text field.

Reply:
Right, so this would be something like "allowAlpha" (or perhaps the reverse, "allowOnlyOpaque"). I'd also suggest the default should be to allow alpha values.

Reply:
SG